en/docs/learn/multi-attribute-login.md

```diff
@@ -0,0 +1,77 @@
+# Multi Attribute Login
+
+WSO2 Identity Server can be configured to use multiple attributes as the login identifier. 
+By default WSO2 Identity server uses the username as the login identifier.  First, you need to
+configure WSO2 identity server for multi attribute authentication. The following section explains
+how to configure this.
+
+## Configuring WSO2 IS for Multi Attribute Login
+
+1. Start the Identity Server and log into the management console using your tenant credentials.
+
+!!! info
+    If you use multiple tenant domains, you need to configure multi attribute login in tenantwise.
+
+2. Click Resident under Identity Providers found in the Main tab and expand the **Account Management** tab.
+
+3. Expand the **Multi Attribute Login** tab and select the **Enable Multi Attribute Login** checkbox.
+
+4. Then add clam URIs which allow for multi attribute login in the given text box.
+
+   ![adding-claims-for-multi-attribute-login](../assets/img/learn/multi-attribute-login/adding-claims-for-multi-attribute-login.png)
+
+5. Add Regular Expression for Allowed Claims. 
+
+   Once you have configured WSO2 Identity Server for multi attribute login, you need to provide regular expression for multi attribute allowed claims. Some claims already have a default regex, if not you need to provide it under the claim edit window.
+   Select claim from claim list and open the Edit window. Then enter the Regex pattern under Regular Expression field.
```

Comment:
Give instructions to access this in the Management Console.

Reply:
fixed: ebaf7a6